### PR TITLE
docs(tracing): Add missing searchable span properties

### PR DESCRIPTION
**Pull request by 0Calories**
## DESCRIBE YOUR PR
Adds several missing properties for span searching to the documentation. Check out https://github.com/getsentry/sentry/pull/74787 to see the PR that adds descriptions for these search properties

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Comment by 0Calories** [comment]
Can I get some eyes on this please?

File: docs/concepts/search/searchable-properties/spans.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ Below is a list of keys and tokens that can be used in the span search:
 
 ### `action`
 
-The type of span action.
+The type of span action, e.g `SELECT` for a SQL span or `POST` for an HTTP span.
```
Comment (lizokm) [inline docs/concepts/search/searchable-properties/spans.mdx:23]:
```suggestion
The type of span action, for example, `SELECT` for a SQL span or `POST` for an HTTP span.
```

File: docs/concepts/search/searchable-properties/spans.mdx [hunk 2]
```diff
@@ -48,6 +48,18 @@ Device class is a synthesized field that's calculated by using device info found
 
 - **Type:** string
 
+### `domain`
+
+General scope of the span’s action, i.e. the tables involved in a `db` span or the host name in an `http` span.
```
Comment (lizokm) [inline docs/concepts/search/searchable-properties/spans.mdx:53]:
```suggestion
General scope of the span’s action, for example, the tables involved in a `db` span or the host name in an `http` span.
```

File: docs/concepts/search/searchable-properties/spans.mdx [hunk 3]
```diff
@@ -90,9 +108,15 @@ The unique ID of a message from the queue or messaging system.
 
 - **Type:** string
 
+### `module`
+
+The Insights module that the span is associated with, e.g `cache`, `db`, `http`, etc.
```
Comment (lizokm) [inline docs/concepts/search/searchable-properties/spans.mdx:113]:
```suggestion
The Insights module that the span is associated with, for example, `cache`, `db`, `http`, and so on.
```

File: docs/concepts/search/searchable-properties/spans.mdx [hunk 4]
```diff
@@ -90,9 +108,15 @@ The unique ID of a message from the queue or messaging system.
...
+
+- **Type:** string
+
 ### `op`
 
-Span operation.
+The operation of the span, e.g `http.client`, `middleware`.
```
Comment (lizokm) [inline docs/concepts/search/searchable-properties/spans.mdx:119]:
```suggestion
The operation of the span, for example, `http.client`, `middleware`.
```

**Review by lizokm — APPROVED** [review]
Just a few small suggestions. Thanks for adding!

**Comment by 0Calories** [comment]
Thanks! Suggestions look good

**Pull request merged**